packages/gluestick/src/config/webpack/webpack.config.client.prod.js

```diff
@@ -4,29 +4,13 @@ import type { WebpackConfig, UniversalWebpackConfigurator } from '../../types';
 
 const webpack = require('webpack');
 const path = require('path');
-const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 module.exports = (
   clientConfig: UniversalWebpackConfigurator,
 ): WebpackConfig => {
   const configuration: Object = clientConfig({ development: false });
   configuration.devtool = 'hidden-source-map';
-  const scssLoaders = configuration.module.rules[1].use;
```

Comment:
This is a bit slower since it runs css through scss-loader, but we're not really doing many css loaders.
ExtractCssChunks doesn't currently do a good job with multiple rulesets for CSS imports.